.github/workflows/elixir.yml

```diff
@@ -1,84 +1,15 @@
 name: Elixir CI
 
-on:
```

Comment:
I suppose there's nothing wrong with current jobs defined in .github/workflows/elixir.yml
The expected steps maybe can be cleaned by repo admin.
https://github.community/t/is-there-a-way-to-delete-or-hide-old-renamed-workflows/16281/20
I also saw a long thread about this being an issue, not sure if it is fixed now.

Reply:
Yes, this was my impression as well. I found [this](https://github.community/t/github-actions-pull-requests-always-include-some-outdated-checks/16157) as well, which looks similar. @alfert please check the settings and feel free to close this PR if the settings fix this issue.

Reply:
There were indeed several checks for master activated which do not fit the current naming scheme in the `elixir.xml` file. I am pretty sure that this is the reason of this annoying task and that we don't have to delete our checks.

Reply:
And yes, after deleting the checks, #182 is green - as it should since only documentation was changed.